src/Dsn.php

```diff
@@ -239,6 +239,18 @@ protected function parseUrlArray($url)
 
         $url = array_merge($this->uriKeys, $url);
 
+        if (isset($url['host'])) {
```

Comment:
Sure. I didn't want to over-complicate things with a loop, but it will certainly keep things DRYer. I'll change them to foreach.